packages/create-carbon/template/authorized.hbs

```diff
@@ -0,0 +1,25 @@
+---
+path: src/events/authorized.ts
+---
+
+import {
+	ApplicationIntegrationType,
+	ApplicationWebhookEventType,
+	type Client,
+	Listener,
+	type ListenerEventData
+} from "@buape/carbon"
+
+export default class ApplicationAuthorizedListener extends Listener {
+	type = ApplicationWebhookEventType.ApplicationAuthorized
+	override async handle(
+		data: ListenerEventData<ApplicationWebhookEventType.ApplicationAuthorized>,
+		client: Client
```

Comment:
Client is unused?

Reply:
Normally the client can be accessed in the CommandInteraction (or similar) classes, because those extend Base, but because there are so many different listener events, I just went with data only here, and then had the client available as a second parameter so it can still be used (as Listeners don't have a client available, just like Commands and Components)

Reply:
I don't have issue with client being passed to listener (although now that the client handling is normal again you can technically export it and use it anyway). I was referring to the client variable being unused in the method, this would result in "'client' is declared but its value is never read. ts(6133)". It's not a huge deal tho as the tsconfig allows for this, so feel free to mark this as resolved.